frontend/templates/product-list/sections/FilterableProductsSection/RefinementMultiSelect.tsx

```diff
@@ -1,13 +1,12 @@
-import { Box, Button, Checkbox, HStack, Text, VStack } from '@chakra-ui/react';
-import { faSort } from '@fortawesome/pro-solid-svg-icons';
+import { Box, Checkbox, HStack, Text, VStack } from '@chakra-ui/react';
 import * as React from 'react';
 
```

Comment:
super nitpicky but if were cleaning stuff up we could get rid of this extra blank line

Reply:
Since this has CR and QA I'm gonna merge as is, but we could do it in the future